front/src/actions/house.js

```diff
@@ -233,6 +233,12 @@ function createActions(store) {
               [house.id]: RequestStatus.RoomValidationError
             }
           });
+        } else if (status === 422 && url.includes('/house')) {
+          store.setState({
+            houseUpdateStatus: {
+              [house.id]: RequestStatus.HouseValidationError
```

Comment:
Is there really a difference with RequestStatus.ValidationError?

Reply:
Yes it's different @atrovato.  
Yes, it is different. Look at the issue I specified in the PR description (#1055). When you fill in the field with a name that is too long, the RequestStatus.ValidationError error message returns that the name of the house cannot be empty.
![image](https://user-images.githubusercontent.com/35010958/106350937-e89d1a00-62d8-11eb-8757-1f52a294bcb9.png)

Reply:
I think you only have to change to current message to inform both information: name is mandatory and max size is N chars.
And maybe limit the name input field to N chars.

On this page, if you empty the name, you will get a 422 too.